iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -601,15 +592,14 @@ LogicalResult mapLinalgOpToLocalInvocationIdImpl<linalg::CopyOp>(
 template <typename LinalgOpTy>
 struct MapLinalgOpToLocalInvocationId : public OpConversionPattern<LinalgOpTy> {
   MapLinalgOpToLocalInvocationId(MLIRContext *context,
-                                 bool usingLinalgOnTensorsPath,
                                  PatternBenefit benefit = 1)
-      : OpConversionPattern<LinalgOpTy>(context, benefit),
-        usingLinalgOnTensorsPath(usingLinalgOnTensorsPath) {}
+      : OpConversionPattern<LinalgOpTy>(context, benefit) {}
+
   LogicalResult matchAndRewrite(
       LinalgOpTy linalgOp, ArrayRef<Value> operands,
       ConversionPatternRewriter &rewriter) const override {
-    if (failed(mapLinalgOpToLocalInvocationIdImpl(linalgOp, operands, rewriter,
-                                                  usingLinalgOnTensorsPath)))
+    if (failed(mapLinalgOpToLocalInvocationIdImpl(
```

Comment:
Actually, could you just follow the call stack here and remove the arguments as its passed down. I think you can remove it everywhere upto the function at line 441 here.

Reply:
Done.